core/deployment/src/main/java/io/quarkus/deployment/annotations/Produces.java

```diff
@@ -0,0 +1,20 @@
+package io.quarkus.deployment.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * The repeatable holder for {@link Produce}.
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.METHOD)
+public @interface Produces {
```

Comment:
Eh, pls no more`@Produces`... we have `javax.enterprise.inject.Produces` and `javax.ws.rs.Produces` already.

Reply:
That's just the repeatable holder.  I've already renamed them to `ProduceRepeatable`, `ConsumeRepeatable`.

Reply:
Not that it matters that much because this API is only visible to extension authors...

Reply:
>  I've already renamed them to ProduceRepeatable, ConsumeRepeatable

+1

> Not that it matters that much because this API is only visible to extension authors...

Even the extension authors might get confused ;-). They will most likely see the CDI API and very often also JAX-RS API.

Reply:
Hah, I told you this was a touchy subject ;)

Reply:
Note that extension authors, when they work on the RESTEasy extension, for example, will already have the JAX-RS and CDI ones in their classpath because the deployment module imports the runtime modules. So yes, this is only visible to extension authors, but given that this PR feature is also only for them, we have to think about them with compassion and caring :)